Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -993,6 +993,32 @@ final class MigrationTests: XCTestCase {
         let newOrderKey = try XCTUnwrap(migratedOrder.value(forKey: "chargeID") as? String)
         XCTAssertEqual(newOrderKey, orderValue)
     }
+
+    func test_migrating_from_63_to_64_enables_creating_new_InboxNote() throws {
+        // Given
+        let sourceContainer = try startPersistentContainer("Model 63")
+        let sourceContext = sourceContainer.viewContext
+
+        try sourceContext.save()
+
+        // When
+        let targetContainer = try migrate(sourceContainer, to: "Model 64")
+
+        // Then
+        let targetContext = targetContainer.viewContext
+        XCTAssertEqual(try targetContext.count(entityName: "InboxNote"), 0)
+
+        // Creates a `InboxNote`
+        let inboxNote = insertInboxNote(to: targetContext
+        )
+        // Creates an `OrderItemAttribute` and adds it to `InboxNote`.
+        let inboxAction = insertInboxAction(to: targetContext)
+        inboxNote.setValue(NSSet(array: [inboxAction]), forKey: "actions")
```

Comment:
nit: maybe we can also assert that the relationship is set using `entity.relationshipsByName`?